tests/test_permissions.py

```diff
@@ -0,0 +1,1254 @@
+    ),
+    field_id('EnvironmentSimplified', 'organization'): TestData(
+        resource_ignore=IgnoreReason.SIMPLIFIED, tenant_ignore=IgnoreReason.NOTREQUIRED
+    ),
+    field_id('Feed', 'messages'): TestData(
+        resource_ignore=IgnoreReason.APPSUPPORT, tenant_ignore=IgnoreReason.NOTREQUIRED
+    ),
+    field_id('Glossary', 'associations'): TestData(
+        resource_ignore=IgnoreReason.PUBLIC, tenant_ignore=IgnoreReason.NOTREQUIRED
+    ),
+    field_id('Glossary', 'categories'): TestData(
+        resource_ignore=IgnoreReason.PUBLIC, tenant_ignore=IgnoreReason.NOTREQUIRED
+    ),
+    field_id('Glossary', 'children'): TestData(
+        resource_ignore=IgnoreReason.PUBLIC, tenant_ignore=IgnoreReason.NOTREQUIRED
+    ),
+    field_id('Glossary', 'stats'): TestData(
+        resource_ignore=IgnoreReason.PUBLIC, tenant_ignore=IgnoreReason.NOTREQUIRED
+    ),
+    field_id('Glossary', 'tree'): TestData(resource_ignore=IgnoreReason.PUBLIC, tenant_ignore=IgnoreReason.NOTREQUIRED),
+    field_id('Glossary', 'userRoleForGlossary'): TestData(
+        resource_ignore=IgnoreReason.PUBLIC, tenant_ignore=IgnoreReason.NOTREQUIRED
+    ),
+    field_id('GlossaryTermLink', 'target'): TestData(
+        resource_ignore=IgnoreReason.PUBLIC, tenant_ignore=IgnoreReason.NOTREQUIRED
+    ),
+    field_id('GlossaryTermLink', 'term'): TestData(
+        resource_ignore=IgnoreReason.PUBLIC, tenant_ignore=IgnoreReason.NOTREQUIRED
+    ),
+    field_id('Group', 'environmentPermissions'): TestData(
+        resource_perm=LIST_ENVIRONMENT_GROUP_PERMISSIONS, tenant_ignore=IgnoreReason.NOTREQUIRED
+    ),
+    field_id('Group', 'tenantPermissions'): TestData(
+        resource_ignore=IgnoreReason.USERROLEINRESOURCE, tenant_ignore=IgnoreReason.TENANT, tenant_admin_perm=True
+    ),
+    field_id('MetadataForm', 'fields'): TestData(
+        resource_ignore=IgnoreReason.INTRAMODULE, tenant_ignore=IgnoreReason.NOTREQUIRED
+    ),
+    field_id('MetadataForm', 'homeEntityName'): TestData(
+        resource_ignore=IgnoreReason.INTRAMODULE, tenant_ignore=IgnoreReason.NOTREQUIRED
+    ),
+    field_id('MetadataForm', 'userRole'): TestData(
+        resource_ignore=IgnoreReason.INTRAMODULE, tenant_ignore=IgnoreReason.NOTREQUIRED
+    ),
+    field_id('MetadataFormField', 'glossaryNodeName'): TestData(
+        resource_ignore=IgnoreReason.PUBLIC, tenant_ignore=IgnoreReason.NOTREQUIRED
+    ),
+    field_id('MetadataFormSearchResult', 'hasTenantPermissions'): TestData(
+        resource_ignore=IgnoreReason.USERROLEINRESOURCE, tenant_ignore=IgnoreReason.TENANT, tenant_admin_perm=True
+    ),
+    field_id('Mutation', 'DisableDataSubscriptions'): TestData(
+        tenant_perm=MANAGE_ENVIRONMENTS, resource_perm=ENABLE_ENVIRONMENT_SUBSCRIPTIONS
+    ),
+    field_id('Mutation', 'addConnectionGroupPermission'): TestData(
+        tenant_ignore=IgnoreReason.BACKPORT, resource_ignore=IgnoreReason.NOTREQUIRED
+    ),
```

Comment:
In resource_ignore, can you change it to BACKPORT? Permissions are required but I do not want to pollute `main` with other PRs

Reply:
Same for createRedshiftConnection

Reply:
Same for deleteConnectionGroupPermission